src/HPWH.hh

```diff
@@ -269,8 +269,9 @@ public:
 
 	/** specifies the unit type for outputs in the CSV file-s  */
 	enum CSVOPTIONS {
-		CSVOPT_NONE,
-		CSVOPT_IPUNITS
+		CSVOPT_NONE = 0,
+		CSVOPT_IPUNITS = 1 << 0,
+		CSVOPT_IS_DRAWING = 1 << 1
```

Comment:
This is a little weird, but since whether the system is drawing or not is a row-level property (not a whole CSV level property). It's not critical at this point to clean up this code, though.